web_src/js/components/RepoActionView.vue

```diff
@@ -809,6 +832,11 @@ export function initRepositoryActionView() {
   user-select: none;
 }
 
+.job-log-line:target > .line-num, .line-num:hover {
+  color: var(--color-primary);
+  text-decoration: underline;
+}
+
```

Comment:
I think you can instead add a `silenced` class to the `<a>` for the same effect.

Reply:
Use `muted` for `a` in https://github.com/go-gitea/gitea/pull/25532/commits/77c550a97b19b54b3d500c6af1f80faa4227f53b, but for `.job-log-line:target > .line-num`, I might still need this.

Reply:
Right, `muted` for color effect, I missed that it had it.

Reply:
Could try a global `a:target` style but I guess it may also have side-effects elswhere, so this is definitely the safer option.